backup_and_restore/application_backup_and_restore/oadp-intro.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="oadp-introduction"]
+include::_attributes/common-attributes.adoc[]
+= Introduction to {oadp-full}
+:context: oadp-api
+:namespace: openshift-adp
+:local-product: OADP
+
+toc::[]
+
+The {oadp-first} product safeguards customer applications on {product-title}. OADP provides disaster recovery protection for {product-title} applications, application-related cluster resources, persistent volumes, and internal images.
+
+OADP can be used to back up containerized applications and virtual machines (VMs).
+
+OADP cannot be used as a disaster recovery solution for etcd or OpenShift Operators. For more information on etcd, see xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[Backing up etcd].
+
```

Comment:
I'm kind of playing around with the idea of combining those three paragraphs (as I did above), and then making a separate section for the APIs.  eg:

```suggestion
// Module included in the following assemblies:
//
// * backup_and_restore/application_backup_and_restore/oadp-intro.adoc

[id="oadp-apis_{context}"]
= {oadp-full} APIs

{oadp-first} provides APIs to enable multiple approaches to customize backups and prevent the inclusion of unnecessary or inappropriate resources. 

OADP provides the following APIs:

* xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#backing-up-applications[Backup]
* xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#restoring-applications[Restore]
* Schedule
* BackupStorageLocation
* VolumeSnapshotLocation


```